inventory/splunk_defaults_linux.yml

```diff
@@ -21,6 +21,7 @@ config:
 
 splunk:
     build_location: "/tmp/splunk.tgz"
+    uf_build_location: "/tmp/splunk.tgz"
```

Comment:
i think these are ok to be default values. Though these variables should be overwritable in environ.py

Reply:
Yes, they are overwritten by local builds and custom build url's.